packages/opentelemetry-node/yarn.lock

```diff
@@ -0,0 +1,30 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```

Comment:
l: Why is this lock file generated in this package? The dependencies in the file don't seem to be package-specific to me 🤔

Reply:
Will `rm -rf`, thanks!